source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -193,7 +194,8 @@
     "API_KEY": "example_api_key",
     "EMAIL_FROM": "admin@example.com",
     "TEMPLATES": {
-        "CONFIRM_SIGN_UP": "example_template_id"
+        "CONFIRM_SIGN_UP": "example_template_id",
+        "CONFIRM_CREATE_PARTICIPATION": "example_template_id",
```

Comment:
We should establish a clean name for all templates. 

I am not even sure for `"CONFIRM_SIGN_UP"`, since a confirmation by the user is needed to activate the account. If the `CONFIRM` keyword is used to represent the fact that a user action is needed to complete the process, that means that `"CONFIRM_CREATE_PARTICIPATION"` needs to provide a link to confirm the subscription to the event.

Maybe we should change `"CONFIRM_SIGN_UP"` to `"ACCOUNT_ACTIVATION"` ?

As for the new template, what would you think about something like `"CONFIRM_EVENT_SUBSCRIPTION"` or even `"CONFIRM_PARTICIPATION"`.

---
My only concern is that the administrator should be able to understand instantly where each template is used.